subsys/dsp/Kconfig

```diff
@@ -32,6 +38,16 @@ config DSP_BACKEND_CUSTOM
 	  Rely on the application to provide a custom DSP backend. The implementation should be
 	  added to the 'zdsp' build target by the application or one of its modules.
 
+config DSP_BACKEND_ARC
+	bool "Use the mwdt library as the math backend"
+	depends on ARCMWDT_LIBC
+	depends on CMSIS_DSP
+	select DSP_BACKEND_HAS_STATIC
+	select DSP_POINTER_HAS_RESTRICT
+	help
+	  Implement the various zephyr DSP functions using the MWDT-DSP library. This feature
+	  requires the MetaWare toolchain and CMSIS module to be selected.
```

Comment:
Why does `DSP_BACKEND_ARC` depend on `CMSIS_DSP`?

My understanding was that Synopsys had their own ARC DSP libraryloosely based on CMSIS-DSP, is that not the case?

Reply:
Synopsys DSP library currently doesn't have corresponding functions like clip(), and(), or(), not(), xor() comparing to CMSIS DSP library. These functions are mapped to CMSIS library temporarily.